internal/xcrd/crd_test.go

```diff
@@ -1164,16 +1152,62 @@ func TestForCompositeResourceClaim(t *testing.T) {
 					},
 				},
 			},
-		},
+		}
 	}
 
-	got, err := ForCompositeResourceClaim(d)
-	if err != nil {
-		t.Fatalf("ForCompositeResourceClaim(...): %s", err)
+	type test struct {
+		name   string
+		schema string
+		want   *extv1.CustomResourceDefinition
```

Comment:
We usually make test cases a map with CamelCase names as keys and call `reason` what you are here setting as `name`. We also define `want` and `args` struct to wrap the desired outcome and provided inputs. Would it be possible to adhere to the standard?

Reply:
I like that pattern.